course/models.py

```diff
@@ -356,7 +356,15 @@ def clean(self):
             errors['lifesupport_time'] = _("Lifesupport time must be later than ending time.")
         if self.archive_time and self.archive_time <= self.lifesupport_time:
             errors['archive_time'] = _("Archive time must be later than lifesupport time.")
-        if not self.is_valid_language(self.language):
+        if self.language.startswith("|"):
+            langs = list(filter(None, self.language.split("|"))) # remove pipes & empty strings
+            for lang in langs:
+                if not self.is_valid_language(lang):
+                    if "language" in errors:
+                        errors['language'] = "%s%s" % (errors['language'], (", " + lang))
+                    else:
+                        errors['language'] = _("Language code(s) missing from settings: " + lang)
```

Comment:
This doesn't go well in translation. Text inside function `_()` is translated, but it's now different based on language. `_("plaa plaa: ") + lang` should be good enough here.